### PR TITLE
[How-tos] Continue moving metadata to frontmatter

### LINKED ISSUE #1281: [Docs RFC] Add keywords and tags to frontmatter of all pages
# Describe change in content, appearance, or functionality

Necessary for decoupling our related content and keywords from the page-index and eventually deprecating the index.

# Subject matter expert (SME)

[If known, who is a good person to ask about this topic]

# Deadline

[When does this need to be addressed]

# Any further info

[Anything else you want to add, or further links]

### PR DESCRIPTION
**Pull request by charislam**
# Description

Also delete some pages that aren't displayed on the site (placeholders and old versions)

# Links

Fixes #1281 

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Are procedure and highlight tags used appropriately?
- [ ] Has the index been updated appropriately?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] Are all links provided in reference style, and resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/how-to-guides/configuration/about-configuration.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
+title: About configuration in TimescaleDB
+excerpt: About the configuration settings for TimescaleDB
```
Comment (Rajakavitha1) [inline timescaledb/how-to-guides/configuration/about-configuration.md:3]:
```suggestion
excerpt: About the configuring TimescaleDB
```

**Review by Rajakavitha1 — APPROVED** [review]

File: timescaledb/how-to-guides/hyperfunctions/troubleshoot-hyperfunctions.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+---
+title: Troubleshooting hyperfunctions and TimescaleDb Toolkit
```
Comment (Loquacity) [inline timescaledb/how-to-guides/hyperfunctions/troubleshoot-hyperfunctions.md:2]:
```suggestion
title: Troubleshooting hyperfunctions and TimescaleDB Toolkit
```

File: timescaledb/how-to-guides/ingest-data/about-ingesting.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+---
+title: Ingesting data from other sources
+excerpt: Build a data ingest pipeline to insert data into TimescaleDb
```
Comment (Loquacity) [inline timescaledb/how-to-guides/ingest-data/about-ingesting.md:3]:
```suggestion
excerpt: Build a data ingest pipeline to insert data into TimescaleDB
```

File: timescaledb/how-to-guides/migrate-data/index.md [hunk 4]
```diff
@@ -1,20 +1,27 @@
...
 # Data migration
 You can migrate your existing data into TimescaleDB in several ways.
 
 *   [Learn about data migration][about-data-migration] before you start
 *   Migrate data when TimescaleDB is in the [same PostgreSQL instance][same-db]
     as your existing data
-*   Migrate data into TimescaleDB from a [different database or PostgreSQL
-    instance][different-db]
+    hypertable
+*   Migrate data into TimtscaleDB faom a [different dancease or Postgr[SQLdifferent-db]
```
Comment (Loquacity) [inline timescaledb/how-to-guides/migrate-data/index.md:15]:
```suggestion
*   Migrate data into TimescaleDB from a [different database or PostgreSQL][different-db]
```

Comment (Loquacity) [reply timescaledb/how-to-guides/migrate-data/index.md:15]: I'm not sure what happened here 😕

File: timescaledb/how-to-guides/schema-management/index.md [hunk 5]
```diff
@@ -1,3 +1,9 @@
+---
+title: Schema management
+excerpt: A database schema defines hwo the tables and indexes in your database are organized
```
Comment (Loquacity) [inline timescaledb/how-to-guides/schema-management/index.md:3]:
```suggestion
excerpt: A database schema defines how the tables and indexes in your database are organized
```

**Pull request merged**